### DISCUSSION TITLE
[Regression] Material UI Checkbox does not get inititalized with a default value

### DISCUSSION THREAD

**Discussion by mak-elena** [react-hook-form/react-hook-form#3224] [Q&A] [answered]
I struggled to initialize an MUI Checkbox properly. so I have tried different approaches recommended in your documentation.

Then I found this pretty simple example:
______________
This is probably a more relevant question for MUI's checkbox, but i just quicky did it with my own understanding.

https://codesandbox.io/s/serverless-dream-jm134?file=/src/App.js

_Originally posted by @bluebill1049 in https://github.com/react-hook-form/react-hook-form/issues/1517#issuecomment-620369642_
______________

It worked fine in the sandbox, but still not in my code.

I did some testing in the sandbox with different versions of react-hook-form and found out that the example works correctly  with the versions till the v. 6.0.2 inclusively, but since the v. 6.0.3  initialization of checkboxes does not work anymore.

**Comment by bluebill1049** [comment]
please use `render` instead `as` for component doesn't support standard props as: `onChange` `onBlur` and `value`.

here is the official codesandbox with lots of examples: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r

**Comment by bluebill1049** [comment, marked as answer]
in case you didn't know where is the offical CSB location: https://react-hook-form.com/api#Controller
<img width="945" alt="Screen Shot 2020-10-21 at 6 29 36 pm" src="https://user-images.githubusercontent.com/10513364/96687280-66dd4c80-13cb-11eb-863e-3ad97f0b9dfe.png">

**Reply by mak-elena** [reply]
Thank you a lot for your prompt answer.
It works perfectly now!

**Reply by bluebill1049** [reply]
np 👍
